### PR TITLE
Lifts of types

### PR DESCRIPTION
**Pull request by EgbertRijke**
In this pull request, dual to #999 I propose a definition of lifts of types.

### CODE REVIEW

**Comment by EgbertRijke** [comment]
Note that there is already some work in `slice` about this definition, and we would face the question whether it is desirable to rephrase type duality in terms of lifts of types.

File: src/foundation/lifts-types.lagda.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+open import foundation.dependent-pair-types
+open import foundation.universe-levels
+```
+
+</details>
+
+## Idea
+
+Consider a type `X`. A {{#concept "lift" Disambiguation="type" Agda=lift-type}}
+of `X` is an object of the slice category of `X`, i.e., it consists of a type
```
Comment (fredrik-bakke) [inline src/foundation/lifts-types.lagda.md:19]:
```suggestion
of `X` is an object in the [slice](foundation.slice.md) over `X`, i.e., it consists of a type
```

**Review by fredrik-bakke — APPROVED** [review]
Sorry for the delayed review. Looks good to me!

File: src/foundation/lifts-types.lagda.md (hunk 1 above)
Comment (fredrik-bakke) [reply src/foundation/lifts-types.lagda.md:19]: (or some other formulation in which a link to the slice file makes sense)

**Comment by fredrik-bakke** [comment]
I'm still a bit confused about how these relate to lifts and extensions of maps, but I don't think that should block these PRs.

**Comment by fredrik-bakke** [comment]
Before I merge, what do you think about coauthorship on these prs?

**Comment by EgbertRijke** [comment]
I’m happy to share. This PR came out of a discussion we had

**Comment by fredrik-bakke** [comment]
Okay, great 😄

**Pull request merged**